components/keepassx/Makefile

```diff
@@ -0,0 +1,41 @@
+#
```

Comment:
Hijacking the discussion, see https://github.com/OpenIndiana/oi-userland/blob/oi/hipster/doc/makefile-targets.txt#L108